makepanda/makepanda.py

```diff
@@ -2862,6 +2862,20 @@ def CreatePandaVersionFiles():
     if "No module named %s" not in str(err):
         raise""" % (module, module)
 
+panda_modules_code += """
+
+from direct.showbase import DConfig
```

Comment:
Fair point, though it also means that the `DConfig` class itself won't be available to those expecting to be able to import it from `pandac.PandaModules` still.  But, it would be a fairly safe bet that nobody is doing that anyway.

I'm personally not too worried about `pandac.PandaModules` importing `direct`; the kind of legacy application that still uses this deprecated module is probably also the kind of legacy application that already uses direct.  But if you suspect this might be an issue, then I am fine with moving the import into the function.

Reply:
Oh I see. `DConfig` is also a class that this module is trying to imitate.

In that case giving the `DConfig` module a `__call__` (which returns itself) might be in order so that old apps can do `DConfig()` - but as you said it's a fairly safe bet nobody's using `DConfig`. Oh well, your call.